config/config_kubernetes.yaml

```diff
@@ -32,7 +32,8 @@ performance_monitoring:
     uuid:                                                 # uuid for the run is generated by default if not set
     enable_alerts: False                                  # Runs the queries specified in the alert profile and displays the info or exits 1 when severity=error
     alert_profile: config/alerts                          # Path to alert profile with the prometheus queries
-
+    check_critical_alerts: False                          # When enabled will check prometheus for critical alerts firing post chaos after soak time for the cluster to settle down
+    critical_alerts_check_soak_time: 120                  # Soak time for the cluster to settle down before checking for critical alerts. Default: 2 mins 
```

Comment:
Maybe we can reuse wait_duration, since critical alerts need to be checked at the end of each chaos run and before the next chaos scenario, we could wait on wait_duration and verify for critical alerts if enabled.

Reply:
Fixed it. Also updated it to run the critical alerts check at the end of each scenario in case multiple scenarios are defined in the config like we discussed. Thanks.